cmd/kube-proxy/app/server.go

```diff
@@ -169,15 +172,19 @@ func AddFlags(options *Options, fs *pflag.FlagSet) {
 }
 
 func NewOptions() (*Options, error) {
+	scheme, codecs, err := proxyscheme.NewSchemeAndCodecs()
+	if err != nil {
+		return nil, err
+	}
+
 	o := &Options{
-		config:      new(componentconfig.KubeProxyConfiguration),
+		config:      new(proxycomponentconfig.KubeProxyConfiguration),
 		healthzPort: ports.ProxyHealthzPort,
+		scheme:      scheme,
+		codecs:      *codecs,
 	}
 
-	o.scheme = runtime.NewScheme()
-	o.codecs = serializer.NewCodecFactory(o.scheme)
-
-	if err := componentconfig.AddToScheme(o.scheme); err != nil {
+	if err := proxycomponentconfig.AddToScheme(o.scheme); err != nil {
```

Comment:
Your `NewSchemeAndCodecs` implementation already adds to the scheme.

Reply:
Done. Thanks!